packages/e2e-tests/specs/editor/blocks/cover.test.js

```diff
@@ -7,11 +15,110 @@ import {
 	openDocumentSettingsSidebar,
 } from '@wordpress/e2e-test-utils';
 
+async function upload( selector ) {
+	await page.waitForSelector( selector );
+	const inputElement = await page.$( selector );
+	const testImagePath = path.join(
+		__dirname,
+		'..',
+		'..',
+		'..',
+		'assets',
+		'10x10_e2e_test_image_z9T8jK.png'
+	);
+	const filename = uuid();
+	const tmpFileName = path.join( os.tmpdir(), filename + '.png' );
+	fs.copyFileSync( testImagePath, tmpFileName );
+	await inputElement.uploadFile( tmpFileName );
+	await page.waitForSelector(
+		`.wp-block-cover img[src$="${ filename }.png"]`
+	);
+	return filename;
+}
+
 describe( 'Cover', () => {
 	beforeEach( async () => {
 		await createNewPost();
 	} );
 
+	it( 'can be created using color picker from the block placeholder', async () => {
```

Comment:
just a nit pick so feel free to ignore - I had to read this a couple of times as my thinking was that it was selecting `Cover` from the block inserter that `created` the block - maybe this could read `overlay color can be set using color picker on block placeholder`

Reply:
That does sound better - will update. Thanks for the suggestion!

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/36321/commits/05a1e39d74c1250db2ad83c7097c8229006c5f63